cri/v1alpha2/cri.go

```diff
@@ -1128,7 +1132,41 @@ func (c *CriManager) UpdateContainerResources(ctx context.Context, r *runtime.Up
 // to either create a new log file and return nil, or return an error.
 // Once it returns error, new container log file MUST NOT be created.
 func (c *CriManager) ReopenContainerLog(ctx context.Context, r *runtime.ReopenContainerLogRequest) (*runtime.ReopenContainerLogResponse, error) {
-	return nil, fmt.Errorf("ReopenContainerLog Not Implemented Yet")
+	containerID := r.GetContainerId()
+
+	container, err := c.ContainerMgr.Get(ctx, containerID)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get container %q with error: %v", containerID, err)
+	}
+	if !container.IsRunning() {
+		return nil, errors.Wrap(errtypes.ErrPreCheckFailed, "container is not running")
+	}
+
+	// get the container's podSandbox id.
+	podSandboxID, ok := container.Config.Labels[sandboxIDLabelKey]
+	if !ok {
+		return nil, fmt.Errorf("failed to get the sandboxId of container %q", containerID)
+	}
+
+	// get logPath of container
+	res, err := c.SandboxStore.Get(podSandboxID)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get metadata of %q from SandboxStore: %v", podSandboxID, err)
+	}
+	sandboxMeta, ok := res.(*SandboxMeta)
+	if !ok {
+		return nil, fmt.Errorf("failed to type asseration for sandboxMeta: %v", res)
+	}
+	logPath, ok := sandboxMeta.ContainerLogMap[containerID]
+	if !ok {
+		logrus.Infof("log path of container: %q is empty", containerID)
```

Comment:
Why did you just print a log here. Empty logPath is not acceptable in `attachCRILog`.
Please also add a precheck at https://github.com/alibaba/pouch/blob/master/daemon/mgr/container.go#L975 
if logPath is empty, immediately return error